controllers/datadogagent/feature/enabledefault/feature.go

```diff
@@ -288,6 +288,17 @@ func (f *defaultFeature) agentDependencies(managers feature.ResourceManagers, co
 		errs = append(errs, err)
 	}
 
+	// Seccomp profiles
```

Comment:
maybe for clarity we can include a comment that this is mounted in the init volume container in the agent default code
```suggestion
	// Create a configmap for the default seccomp profile in the System Probe. This is mounted in the init-volume container in the agent default code.
```